docs/2.guide/2.directory-structure/1.server.md

```diff
@@ -315,8 +315,8 @@ export default defineEventHandler((event) => {
 ### Sending Redirect
 
 ```ts [server/api/foo.get.ts]
-export default defineEventHandler((event) => {
-  return sendRedirect(event, '/path/redirect/to', 302)
+export default defineEventHandler(async (event) => {
+  await sendRedirect(event, '/path/redirect/to', 302)
```

Comment:
Would be nice idea to return + await (only for readability). This way we can chain the result and be upward compatible.